### PR TITLE
[MRG+1] Fix dipole amplitude + plotter.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/4049

### CODE REVIEW

File: mne/beamformer/_rap_music.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def _make_dipoles(times, poss, oris, sol, gof):
     dipoles : list
         The list of Dipole instances.
     """
-    amplitude = sol * 1e9
+    amplitude = sol
```
Comment (jaeilepp) [inline mne/beamformer/_rap_music.py]: Are we happy with doing this without deprecation?

Comment (joewalter) [inline mne/beamformer/_rap_music.py]: Why are you keeping amplitude, and not just using sol in Dipole(). Saves a few lines. 

In my opinion this is a bugfix not a feature change, as e.g. saving the dipoles created with rap music would multiplicate twice by 1e9.

**Comment by joewalter** [comment]
plot_dipole_amplitudes needs a fix, too (dip.amplitude * 1e9), as it plots dipole amplitudes in nAm.

File: mne/beamformer/_rap_music.py (hunk 1 above)
Comment (agramfort) [inline mne/beamformer/_rap_music.py]: +1 to use sol directly. +1 for bug fix too.

**Comment by jaeilepp** [comment]
Comments addressed.

File: mne/viz/misc.py [hunk 2]
```diff
@@ -607,7 +607,7 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
         xlim[1] = max(xlim[1], dip.times[-1])
     ax.set_xlim(xlim)
     ax.set_xlabel('Time (sec)')
-    ax.set_ylabel('Amplitude (nAm)')
+    ax.set_ylabel('Amplitude (Am)')
```
Comment (agramfort) [inline mne/viz/misc.py]: we should still plot amps in nAm by doing the scaling in the plot like we do for Evoked. That's the good unit for looking at data

Comment (joewalter) [inline mne/viz/misc.py]: I'd prefer to keep nAm here for readability, but multiply dipole.amplitude with 1e9 in line 605.

**Comment by Eric89GXL** [comment]
Agreed that this is a bug fix. +1 for merge (if this is indeed MRG)

**Comment by jaeilepp** [comment]
Yes it's MRG, if the Travis builds succeed some day...

**Comment by agramfort** [comment]
thx @jaeilepp

**Pull request merged**